pom.xml

```diff
@@ -51,15 +51,14 @@
 
 	<repositories>
 		<repository>
-			<id>Eclipse 4.16 (2020-06)</id>
+			<id>Eclipse 4.18 (2020-12)</id>
 			<layout>p2</layout>
-			<!--<url>http://download.eclipse.org/releases/2019-09</url> -->
-			<url>http://ftp-stud.hs-esslingen.de/pub/Mirrors/eclipse/releases/2020-06/</url>
```

Comment:
I don’t know why we didn’t use the official update site, whoever made the change left no comment. Since it works with the official site, I think it makes sense to switch to it. This will also make the CI use closer mirrors instead of requesting the dependencies over the Atlantic from a small Swabian town :D

Reply:
The reason for using the mirror was that at least some years ago the official Eclipse updatesite was not available quite often. Thus building against the mirror was more stable. But I have also removed the mirror in some other projects and I am optimistic that availability of the updatesite is better now.